packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -79,15 +87,15 @@
     <_WpfCommonNetFxReference Include="System.Windows.Controls.Ribbon" Condition="'$(_TargetFrameworkVersionWithoutV)' >= '4.5'" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETFramework' And '$(UseWPF)' == 'true' ">
+  <ItemGroup Condition=" ('$(DisableImplicitFrameworkReferences)' != 'true') And ('$(TargetFrameworkIdentifier)' == '.NETFramework') And ('$(_TargetFrameworkVersionWithoutV)' != '') And ('$(_TargetFrameworkVersionWithoutV)' >= '3.0') And ('$(UseWPF)' == 'true') ">
     <_SDKImplicitReference Include="@(_WpfCommonNetFxReference)" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETFramework' And '$(UseWindowsForms)' == 'true' ">
+  <ItemGroup Condition=" ('$(DisableImplicitFrameworkReferences)' != 'true') And ('$(TargetFrameworkIdentifier)' == '.NETFramework') And ('$(_TargetFrameworkVersionWithoutV)' != '') And ('$(_TargetFrameworkVersionWithoutV)' >= '3.0') And ('$(UseWindowsForms)' == 'true') ">
```

Comment:
WinForms is supported on .NET Framework 1.0 onwards. Should this check be converted into `('$(_TargetFrameworkVersionWithoutV)' >= '3.0')` instead? 

That would mean that SDK style projects, specifically WindowsDesktop SDK, is supported for building .NET Framework 1.0 projects. Is this something we want to support (or _enable_, even if it were _unsupported_)? 

/cc @merriemcgaw, @nguerrera

Reply:
I'm confused, do you mean `('$(_TargetFrameworkVersionWithoutV)' >= '1.0')`? It looks like you already have `('$(_TargetFrameworkVersionWithoutV)' >= '3.0') `

>That would mean that SDK style projects, specifically WindowsDesktop SDK, is supported for building .NET Framework 1.0 projects

Are we supporting any .NET framework projects building with the WindowsDesktop SDK?

Reply:
Yes netfx projects are built with WindowsDesktop SDK. It’s supported.

Reply:
Good catch, I did mean to ask about >= 1.0